api/v3/openapi.yaml

```diff
@@ -193,13 +193,13 @@ paths:
       tags:
         - ACL (v3)
       parameters:
-        - $ref: '#/components/parameters/AclResourceType'
```

Comment:
@AndrewJSchofield how did you find the inconsistencies/mistakes in the openapi.yaml? Is there a linter to do it? or good old hard-core eye-balling the code/docs?
Its the former, then we can look to automate it.

Reply:
A combination of JIRAs, reading the spec, running the code and having beady eyes.